epictrack-api/src/api/models/work_status.py

```diff
@@ -27,9 +30,14 @@ class WorkStatus(BaseModelVersioned):
     id = Column(Integer, primary_key=True, autoincrement=True)
     description = Column(String(2000), nullable=False)
     notes = Column(Text)
-    posted_date = Column(Date, nullable=False)
+    posted_date = Column(DateTime(timezone=True), nullable=False)
     posted_by = Column(String(100), nullable=True)
     work_id = Column(ForeignKey('works.id'), nullable=False)
     work = relationship('Work', foreign_keys=[work_id], lazy='select')
     is_approved = Column(Boolean(), default=False, nullable=False)
     approved_by = Column(String(255), default=None, nullable=True)
+
+    @classmethod
+    def list_statuses_for_work_id(cls, work_id) -> List[WorkStatus]:
```

Comment:
I think I'm noticing changes from https://github.com/bcgov/EPIC.track/pull/1195. Should we turn this PR into draft and wait until the other one is merged?